aws-predictions/src/main/java/com/amplifyframework/predictions/aws/request/AWSPollyRequest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.aws.request;
+
+import androidx.annotation.NonNull;
+
+import com.amplifyframework.predictions.aws.models.VoiceType;
+
+import java.util.Objects;
+
+/**
+ * Simple request instance for speech synthesis operation.
+ */
+public final class AWSPollyRequest {
```

Comment:
Don't forget to do equals, hashCode, toString on all of your models, request and response types.

Reply:
After this PR + dummy transcribe PR, I will go through every model to add them!